spanner/internal/benchwrapper/proto/spanner.proto

```diff
@@ -15,31 +15,42 @@
 syntax = "proto3";
 
 package spanner_bench;
+option go_package = "spanner_bench";
 
-message ReadQuery{
-  // The query to use in the read call.
-  string Query = 1;
+
+message Singer {
+  int64 id = 1;
+  string first_name = 2;
+  string last_name = 3;
+  string singer_info = 4;
+}
+
+message Album {
+  int64 id = 1;
+  int64 singer_id = 2;
+  string album_title = 3;
 }
 
-message User{
-  string name = 1;
-  int64 age = 2;
+message ReadQuery {
+  // The query to use in the read call.
+  string query = 1;
 }
 
-message InsertQuery{
-   // The query to use in the insert call.
-  repeated User users = 1;
+message InsertQuery {
+  // The query to use in the insert call.
+  repeated Singer singers = 1;
+  repeated Album albums = 2;
 }
 
-message UpdateQuery{
+message UpdateQuery {
   // The queries to use in the update call.
-  repeated string Queries = 1;
+  repeated string queries = 1;
 }
 
-message EmptyResponse{
-}
+// TODO(deklerk): Replace with Google's canonical Empty.
```

Comment:
why do we add this TODO as deklerk is not working on this any more?

Reply:
Oh I copied this proto from the one in google3, which had this todo, I've removed it now.